Domain/Enums/EBookFormat.cs

```diff
@@ -0,0 +1,17 @@
+using System.ComponentModel;
+
+namespace Domain {
+    public enum EBookFormat
+    {
+        [Description(".epub")]
```

Comment:
assign numbers explicitly

Reply:
Do I need to use description attribute here or it's useless?

Reply:
I have no idea why you added it here, maybe there is some logic behind it